### ISSUE TITLE
Feature request: An option to display return section as a table

### ISSUE THREAD

**Issue by ilyub** [tgreyuk/typedoc-plugin-markdown#227]
Is it possible to add an option to display return this way:
```
#### Returns

| Type | Description |
| :------ | :------ |
| string | - Description of the returned value.
```

**Comment by tgreyuk** [comment]
@ilyub happy to look at improving this but not sure about adding an additional option. Have played around with this and the table layout doesn't work so well for returned object or functions? 

This would achieve horizontal alignment without using a table?

```
#### Returns

`string` Description of the returned value.
```

Would this work for you or is there is a specific use-case for a table?

**Comment by ilyub** [comment]
Hi,

My problem was that my md viewer displayed ####... and **...** differently. So it all looked a bit disordered. But after uploading to github pages and selecting the right theme, it looks good.

Still, adding an option that you suggest above may be useful.
If returns section starts with h4 title (#### Returns) then it is guaranteed that it looks exactly the same as other titles regardless of the theme.
I think you are right that table layout is accessive here.
 
Thank you for your plugin. It generates much more compact docs with no random hashes, so I can include it with repository. Also, compatible with github themes.

**Comment by tgreyuk** [comment]
Thanks for the feedback! I will at a minimum implement the heading style #### for the returns.

**Comment by tgreyuk** [comment]
@ilyub if you are interested I have a pr #229  that aims to improve the readability of the ui in general and also updates 'Returns' heading. Would be useful to know if you think this makes sense.

**Comment by tgreyuk** [comment]
As discussed returns is now in an h4 but not in a table layout. Fix as part of #230.

**Comment by ilyub** [comment]
Thank you. I'll check the change as soon as typedoc adds support for typescript 4.3 (https://github.com/TypeStrong/typedoc/issues/1589). Waiting for them.
